R/test-files.R

```diff
@@ -150,10 +163,18 @@ test_files <- function(test_dir,
                        env = NULL,
                        stop_on_failure = FALSE,
                        stop_on_warning = FALSE,
+                       wrap = lifecycle::deprecated(),
```

Comment:
Just make this `TRUE` and then warn if `wrap` is `FALSE`

Reply:
Done.